strings.json

```diff
@@ -47,52 +58,69 @@
     "en": "Omit Cash from Percentage Returns"
   },
   "accessibility_realizedChange": {
-    "en": "Realized Change"
+    "en": "Realized Change",
+    "zh-Hant-TW": "已實現損益"
   },
   "accessibility_realizedChangePercent": {
-    "en": "Realized Change Percent"
+    "en": "Realized Change Percent",
+    "zh-Hant-TW": "已實現獲利率"
   },
   "accessibility_realizedYTDChange": {
-    "en": "Realized YTD Change"
+    "en": "Realized YTD Change",
+    "zh-Hant-TW": "已實現年化報酬"
   },
   "accessibility_realizedYTDChangePercent": {
-    "en": "Realized YTD Change Percent"
+    "en": "Realized YTD Change Percent",
+    "zh-Hant-TW": "已實現年化報酬率"
   },
   "accessibility_realizedCostBasis": {
-    "en": "Realized Cost Basis"
+    "en": "Realized Cost Basis",
+    "zh-Hant-TW": "投資成本"
```

Comment:
Is '已實投資成本' a better translation?

Reply:
It depends how it show. If it's down below Realized, then can just translate "投資成本" not needy to translate "Realized". But if it's single line, it could be translate into "已實現投資成本"

Reply:
Since this is "accessibility_realizedCostBasis", it means it's for accessibility (people who can't see clearly). So it might be better to be a bit more descriptive. This text will be read using audio to the user (it is not displayed as text in the UI).